$DragonFly: src/lib/libmagic/Attic/file.h.no_obj.patch,v 1.1.2.2 2007/03/25 17:08:05 dillon Exp $
--- file.h.orig	2007-03-24 22:33:12.000000000 -0400
+++ file.h	2007-03-24 22:33:35.000000000 -0400
@@ -51,7 +51,9 @@
 /* Do this here and now, because struct stat gets re-defined on solaris */
 #include <sys/stat.h>
 
+#ifndef COMPILE_ONLY
 #define ENABLE_CONDITIONALS
+#endif
 
 #ifndef MAGIC
 #define MAGIC "/etc/magic"
